content/about/contribute/diagrams/index.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
I think we need to state an important requirement. SVG is typically an output format and we need to have the original source of diagrams. So we can add something about:

1) Put the original images in our team drive: https://drive.google.com/corp/drive/u/0/folders/1jczscJueUBR3IOvH30q9HAZX0jC7GSyW

2) Put a comment in the markdown with the URL to the original source.

Reply:
Along the same lines, it would be considerably more useful, at least for Googlers using Google Drawing, if the sample content were supplied as a Google drawing rather than an SVG.

Reply:
+1 on both of these

Reply:
@geeknoid @frankbu I agree but SVG is an editable format, not a read-only format. I already have a Google drawing version of the guide ready. I can add it to the community drive and add a link.
However, it would be better for people planning on reusing the assets to do so with a proper vector graphics editing tool, such as InkScape, using SVG.

Reply:
@geeknoid What I want to avoid is having two sources of truth for diagrams, the Gdrive and the repo. IMO, only the repo should be the source of truth for content. The guidelines accomplish that by requiring the diagrams be added to the repo in SVG format with editable text but I am open to other formats.

Reply:
Most of our diagrams have been created using Google Drawing, which doesn't support SVG as an input format. I think we need to recognize the reality of the tools most of our developers use and are familiar with.

I agree that having source content in one format and published content be in a different format isn't ideal. But it has generally been working for us, so it's a manageable strategy.

Reply:
@frankbu @geeknoid I've added the guide to the community's Misc folder and a link to it in the doc. Google Draw allows you to export your drawing as an SVG which means contributors can include their Google Draw diagrams in their pull requests as SVG very easily.
PTAL
@geeknoid Any idea how to solve the spellchecker failure due to the SVG quotation marks?